Mathlib/AlgebraicTopology/SimplicialSet/Subcomplex.lean

```diff
@@ -63,6 +63,78 @@ abbrev ι (A : Subcomplex X) : Quiver.Hom (V := SSet) A X := Subpresheaf.ι A
 instance (A : X.Subcomplex) : Mono A.ι :=
   inferInstanceAs (Mono (Subpresheaf.ι A))
 
+section
+
+variable {S₁ S₂ : X.Subcomplex} (h : S₁ ≤ S₂)
+
+/-- Given an inequality `S₁ ≤ S₂` between subcomplexes of a simplicial set,
+this is the induced morphism in the category `SSet`. -/
+abbrev homOfLE : (S₁ : SSet.{u}) ⟶ (S₂ : SSet.{u}) := Subpresheaf.homOfLe h
+
+@[reassoc]
+lemma homOfLE_comp {S₃ : X.Subcomplex} (h' : S₂ ≤ S₃) :
+    homOfLE h ≫ homOfLE h' = homOfLE (h.trans h') := rfl
+
+variable (S₁) in
+@[simp]
+lemma homOfLE_refl : homOfLE (by rfl : S₁ ≤ S₁) = 𝟙 _ := rfl
+
+@[simp]
+lemma homOfLE_app_val (Δ : SimplexCategoryᵒᵖ) (x : S₁.obj Δ) :
+    ((homOfLE h).app Δ x).val = x.val := rfl
+
+@[reassoc (attr := simp)]
+lemma homOfLE_ι : homOfLE h ≫ S₂.ι = S₁.ι := rfl
+
+instance mono_homOfLE : Mono (homOfLE h) := mono_of_mono_fac (homOfLE_ι h)
+
+/-- This is the isomorphism of simplicial sets corresponding to
+an equality of subcomplexes. -/
+@[simps]
+def isoOfEq (h : S₁ = S₂) : (S₁ : SSet.{u}) ≅ S₂ where
```

Comment:
Should this be called `eqToIso` to mimic `CategoryTheory.eqToIso`? 

Reply:
Thanks for the suggestion!